docs/manual/java/guide/cluster/ReadSideRDBMS.md

```diff
@@ -2,76 +2,21 @@
 
 This page is specifically about Lagom's support for relational database read-sides.  Before reading this, you should familiarize yourself with Lagom's general [[read-side support|ReadSide]].
```

Comment:
In order to avoid breaking navigation from older versions, I kept this page as an overview of the two different RDBMS options.

I haven't yet made an equivalent change on the Scala side, because we only support JDBC there still, but should we preemptively rename the page to `ReadSideJDBC.md`?